int32 id
string material
float64 pos_x
float64 pos_y
float64 pos_z
float64 yaw_angle